base/workflow/R/runModule.run.write.configs.R

```diff
@@ -7,6 +7,12 @@
 #' @export
 runModule.run.write.configs <- function(settings, overwrite = TRUE) {
 
+ensemble_size <- settings[[1]]$ensemble$size
+input_design <- PEcAn.uncertainty::generate_joint_ensemble_design(settings=settings[[1]],ensemble_size=ensemble_size)
```

Comment:
are you sure the [[1]] will work as expected if settings is just a single site, not a MultiSettings?